examples/chart/teleport-demo/docker/namespace-cleaner/Dockerfile

```diff
@@ -0,0 +1,19 @@
+FROM quay.io/gravitational/debian-grande:latest
+
+ARG KUBECTL_VERSION="v1.12.4"
```

Comment:
Nitpick: 1.12.5 contains a security fix. I am pretty sure it is not relevant here but will be good to update, anyway.